internal_transfers/actions/src/simulate/models.ts

```diff
@@ -0,0 +1,13 @@
+import { EventLog } from "../models";
+
+/**
+ *  Represents the relevant data returned from a transaction-simulation.
+ */
+export interface SimulationData {
+  // # Transaction hash that would have been assigned if this were an actually mined tx.
+  txHash: string;
+  // Block on which the simulation was made.
+  blockNumber: number;
+  // Event Logs emitted within the transaction's simulation.
+  logs: EventLog[];
+}
```

Comment:
im a bit confused about the structure of the project. I see a pythion project with a nodejs project inside a folder (`internal_tranfer/actions`). And then there's models in the `src` but also in `src/simulate` 

For me, `SimulationData` should be part of the interface of the method `parseSimulation` so it should be in the same file.

Reply:
Yea. I just don't understand how to put function signatures into an interface.. until now I have only seen interfaces used to define data structures (without functionality).

Reply:
> im a bit confused about the structure of the project. I see a pythion project with a nodejs project inside a folder (internal_tranfer/actions).

Yes, what you are seeing is what it is.

> And then there's models in the src but also in src/simulate

models inside of simulate are related only to the simulation abstraction (I thought this would keep simulation self contained). I can also easily move `SimulationData` into the same file (no problem).